addons/apecloud-mysql/scripts/etcd-post-start.sh

```diff
@@ -1,28 +1,42 @@
 #!/bin/bash
-etcd_port=${ETCD_PORT:-'2379'}
-etcd_server=${ETCD_SERVER:-'127.0.0.1'}
-
+endpoints=${ETCD_SERVER:-'127.0.0.1:2379'}
 cell=${CELL:-'zone1'}
-export ETCDCTL_API=2
 
-etcdctl --endpoints "http://127.0.0.1:${etcd_port}" get "/vitess/global" >/dev/null 2>&1
-if [[ $? -eq 1 ]]; then
-  exit 0
-fi
+servers=""
+IFS=',' read -ra ADDR <<< "$endpoints"
+for addr in "${ADDR[@]}"; do
+  if [[ $addr != http* ]]; then
+    addr="http://$addr"
+  fi
+  servers="${servers},${addr}"
+done
+
+servers=${servers:1}
 
-echo "add /vitess/global"
-etcdctl --endpoints "http://127.0.0.1:${etcd_port}" mkdir /vitess/global
+if [[ ${ETCDCTL_API} -eq "3" ]]; then
+  output=$(etcdctl --endpoints="${servers}" get "/vitess/${KB_CLUSTER_NAME}/global" --prefix --keys-only)
```

Comment:
cluster name is not neccessary to be UNIQUE k8s cluster-wide.
maybe use namespace AND clustername.
Check them all.